### PR TITLE
Add EML attachments

### PR DESCRIPTION
**Pull request by gal-forer**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [x] In Progress
- [ ] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
https://jira-hq.paloaltonetworks.local/browse/XSUP-23616

## Description
Added the option of having EML as attachments.

## Screenshots
Paste here any images that will help the reviewer

## Minimum version of Cortex XSOAR
- [ ] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [ ] No

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py [hunk 1]
```diff
@@ -409,7 +433,7 @@ def generate_search_query(time_to_fetch_from: Optional[datetime],
     return messages_query_list
 
 
-def test_module(client: IMAPClient) -> str:
+def script_test_module(client: IMAPClient) -> str:
```
Comment (gal-forer) [inline Packs/MailListener/Integrations/MailListenerV2/MailListenerV2.py:436]: I changed thew name because if it starts with test_ the IDE will think it's a unit test

File: Packs/MailListener/ReleaseNotes/1_0_37.md [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### Mail Listener v2
+
+- Added the ability to have EML files as attachments.
```
Comment (DeanArbel) [inline Packs/MailListener/ReleaseNotes/1_0_37.md:6]: Let's rephrase it to better reflect it's a fix.
```suggestion
- Fixed an issue where EML attachments were not extracted.
```

**Review by DeanArbel — APPROVED** [review]
Good job
